core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java

```diff
@@ -675,6 +675,7 @@ public void userFileBufferBytesOverFlowException() {
   public void shortMasterHeartBeatTimeout() {
     mConfiguration.set(PropertyKey.MASTER_STANDBY_HEARTBEAT_INTERVAL, "5min");
     mConfiguration.set(PropertyKey.MASTER_HEARTBEAT_TIMEOUT, "4min");
+    mConfiguration.set(PropertyKey.MASTER_TTL_CHECKER_INTERVAL_MS, "0");
```

Comment:
please add a separate UT for your case. This UT is testing for `shortMasterHeartBeatTimeout`.

Reply:
yes, thank you.